proof/invariant-abstract/BCorres_AI.thy

```diff
@@ -172,9 +172,9 @@ lemma throw_on_false_bcorres[wp]:
 
 crunch_ignore (bcorres) (add: getActiveIRQ)
 
-lemma preemption_point_bcorres[wp]:
+(* lemma preemption_point_bcorres[wp]:
```

Comment:
@mktnk3 we don't need `bcores` for MCS, right?

Reply:
I think this topic came up a while ago, but I am not sure of how we concluded. I don’t think we can simply get rid of `bcorres` altogether. But, since the extended part of the abstract state is much smaller in MCS, the role of `bcorres` is smaller and hence we probably don’t need a fair part of those files. We’ve removed some part when we worked on AInvs, but I don’t think it is in a fully cleaned state. The fact that we commented out this lemma and AInvs is checking fine probably means we can remove this lemma for MCS.

Reply:
I thought I may as well have a quick look and see if I can prove it. It was really easy. So may as well include it (I'll update the PR soonish with some new commits)